index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobily First</title>
+  <!-- 
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+    rel="stylesheet"
+    />
+
+  -->
+    <link rel="stylesheet" href="style.css">
+  
+    
+
+      <script
+        src="https://kit.fontawesome.com/26c38390b4.js"
+        crossorigin="anonymous"
+        ></script>
+  
+  </head>
+  <body>
+    
+      <nav class="navbar">
+        <p class="brand-title">My logo</p>
+        <i class="fas fa-bars"></i>
+      </nav>
+     
+      <section class="presentation">
+        <h2>I’m Alexandra. Glad to see you!</h2>
+      </section>
+
+      <article class="job">
```

Comment:
- [ ] As a header of your page , kindly use  the `header` tag , 
- [ ] Noe that the header contains the site's title or logo (or both) and the main site navigation 👍 , kindly update that.